sample-app/src/main/java/com/lavanya/aerogearlibrary/MyApolloClient.kt

```diff
@@ -21,7 +21,7 @@ class MyApolloClient : Application() {
     companion object {
 
         //BASE_URL is of the format http://your_ip_adress:4000/graphql
-        private val BASE_URL = "http://192.168.43.243:4000/graphql"
+        private val BASE_URL = "http://192.168.0.106:4000/graphql"
```

Comment:
Kinda antipattern to have it in the code. 
Ignore that now but best to use something like: https://stackoverflow.com/questions/5806220/how-to-connect-to-my-http-localhost-web-server-from-android-emulator-in-eclips/5806384

Reply:
I will change this in the next pr 👍